src/Tribe/Service_Providers/Shortcodes.php

```diff
@@ -22,12 +22,13 @@ public function register() {
 			return;
 		}
 
-		$this->container->singleton( Manager::class, Manager::class );
+		$this->container->singleton( 'shortcode.manager', Manager::class );
```

Comment:
```suggestion
		$this->container->singleton( 'shortcode.manager', Manager::class );
		$this->container->singleton( Manager::class, Manager::class );
```

Reply:
Should we keep it compatible with the other way?

Reply:
Should we just use the class name here so it's still using the same singleton instance?

Reply:
Binding the objects this way will create 2 singleton instances, one for `Manager::class` and one for `shortcode.manager`.

This code will ensure only one singleton instance will be served from both `Manager::class` and `shortcodes.manager`:

```php
$this->container->singleton( Manager::class, $manager );
$this->container->singleton( 'shortcode.manager', function(){
    return $this->container->make( Manager::class );
} );
```

The approach is a bit verbose (issue potentially mitigated pr by [this PR](https://github.com/moderntribe/tribe-common/pull/1392)), but it would ensure both singleton and identity.